interactive/dashboard/flow_diagram.py

```diff
@@ -119,153 +94,1008 @@ def name_included_reactions(included_reactions, reactions_json):
         names_dict.update({reaction_index: name})
     return names_dict
 
+
+# check if element is in 'blocked' elements list
 def isBlocked(blocked, element_or_reaction):
     for bl in blocked:
         if bl == element_or_reaction and len(blocked) != 0:
             return True
     return False
+
+
+# make reaction hot hot hot by adding some nicer arrows and cleaning it up
 def beautifyReaction(reaction):
     if '->' in reaction:
         reaction = reaction.replace('->', ' → ')
     if '_' in reaction:
         reaction = reaction.replace('_', ' + ')
     return reaction
-# generates dict with list of edges and list of nodes: di = {'edges': [], 'species_nodes': [], r_nodes: []}
-def find_edges_and_nodes(contained_reactions, reactions_names_dict, reactions_json, widths_dict, blocked):
-    edges = {}
-    s_nodes = {}
-    r_nodes = []
 
-    edge_colors = []
+
+# undo beautifyReaction (usually used when indexing dictionaries)
+def unbeautifyReaction(reaction):
+    if '→' in reaction:
+        reaction = reaction.replace(' → ', '->')
+    if '+' in reaction:
+        reaction = reaction.replace(' + ', '_')
+    return reaction
+
+
+# return list of species, reactions, species size and colors.
+def findReactionsAndSpecies(list_of_species, reactions_data, blockedSpecies):
+    global minAndMaxOfSelectedTimeFrame
+    global userSelectedMinMax
+    global previous_vals
```

Comment:
These three don't look to be used or modified in this function. Can we delete them?